### PR TITLE
feat: Transaction Details 

### LINKED ISSUE #309: 🚀 [Feature] Transaction details design
## 🚀 Feature

The transaction details which are expandable on the transaction list are very badly designed and need a general overhaul.

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
decay on transaction and transactionlist

### Issues
- fixes #309

### CODE REVIEW

File: frontend/src/components/DecayInformation.vue [hunk 1]
```diff
@@ -11,9 +48,25 @@ export default {
...
+      duration: {},
+      diff: {},
+    }
+  },
+  methods: {
+    getDuration(start, end) {
+      this.a = new Date(start)
+      this.b = new Date(end)
+      this.a = this.$moment.unix(this.a)
+      this.b = this.$moment.unix(this.b)
```
Comment (ulfgebhardt) [inline frontend/src/components/DecayInformation.vue:66]: Wäre mir lieb, wenn du Variablen mit sinnigen Namen benennst - a und b sind nicht aussagekräftig

File: community_server/src/Controller/AppRequestsController.php [hunk 2]
```diff
@@ -393,6 +393,16 @@ public function listTransactions($page = 1, $count = 25, $orderDirection = 'ASC'
...
     
+    public function getDecayStartBlock()
+    {
+        $transactionsTable  = TableRegistry::getTableLocator()->get('Transactions');
+        $decayStartBlock = $transactionsTable->find()->where(['transaction_type_id' => 9]);
+        if(!$decayStartBlock->count()) {
+            return $this->returnJson(['state' => 'error', 'msg' => 'not found']);
+        }
+        return $this->returnJson(['state' => 'success', 'decay_start' => $decayStartBlock->first()->received]);
+    }
```
Comment (ulfgebhardt) [inline community_server/src/Controller/AppRequestsController.php:414]: The docu for this call is missing. Same applies to the changed call returning flag for decay start. If possible please include this in the API docu <3

File: frontend/src/components/DecayInformation.vue [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<template>
+  <div v-if="decay">
```
Comment (Mogge) [inline frontend/src/components/DecayInformation.vue:2]: Do this in the parent component. `decay` should always be a valid decay object

File: frontend/src/components/DecayInformation.vue [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      b: 0,
+      duration: {},
+      diff: {},
+    }
+  },
+  methods: {
+    getDuration(start, end) {
+      this.a = new Date(start)
+      this.b = new Date(end)
+      this.a = this.$moment.unix(this.a)
```
Comment (Mogge) [inline frontend/src/components/DecayInformation.vue:65]: something like
`this.startDate = this.$moment(new Date(start))`

File: frontend/src/views/Pages/AccountOverview/GddTransactionList.vue [hunk 5]
```diff
@@ -17,52 +17,42 @@
...
+                <span class="display-4">{{ item.name }}</span>
+              </div>
+              <div class="display-5">
+                {{
+                  item.type === 'receive' ? 'Nachricht vom Absender:' : 'Nachricht an Empfänger:'
+                }}
+              </div>
+              <div class="display-4">{{ item.memo }}</div>
+
+              <decay-information :decay="getTransaction(item.transaction_id).decay" form="long" />
```
Comment (Mogge) [inline frontend/src/views/Pages/AccountOverview/GddTransactionList.vue:54]: Here `v-if="decay"`

**Review by Mogge — APPROVED** [review]
Megre it, later we check for bugs and make some refactoring

**Pull request merged**